src/styles/_catalog.scss

```diff
@@ -1,22 +1,38 @@
+$cardWidth: 200px;
+$cardHeight: 300px;
+$cardGap: 48px;
+
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  column-gap: $cardGap;
+  row-gap: $cardGap;
+  padding-bottom: 48px;
```

Comment:
do you need this?
```suggestion
```

Reply:
@SerhiiKirik  If I delete it, all items will be without a free place below, or should I just find another decision for this issue? 
![image](https://user-images.githubusercontent.com/125828769/222503240-4fb18211-3c3c-4e15-bff6-54b833978473.png)